mmdet/models/backbones/hrnet.py

```diff
@@ -218,6 +218,8 @@ class HRNet(BaseModule):
             memory while slowing down the training speed.
         zero_init_residual (bool): whether to use zero init for last norm layer
             in resblocks to let them behave as identity.
+        multiscale_output (bool): whether to output multi-level features. If
+            False, only the first level feature will be output. Default: True.
```

Comment:
Add unittests for this `arg`.

Reply:
level --> branch?

Reply:
Add more details for extra `args`.

Reply:
> level --> branch?

should use level. Multi-level features produced by multiple branches

Reply:
> > level --> branch?
> 
> should use level. Multi-level features produced by multiple branches

So it's better to explain it.

> Multi-level features produced by multiple branches